### DISCUSSION TITLE
Minimum version 11.0 on MacOS

### DISCUSSION THREAD

**Discussion by cbertelli** [orbitalquark/textadept#563] [General]
Hello, is there something that prevents building/running on 10.13 to 11.15 versions on MacOS? I would very much like to use version Textadept v. 12 on Mojave.

**Comment by paaguti** [comment]
+1 I need spare cycles to try to see how to compile TA12 from the source code on my High Sierra… but that looks quite like science fiction:(Enviado desde mi iPhoneEl 9 ago 2023, a las 16:13, Carlo Bertelli ***@***.***> escribió:﻿
Hello, is there something that prevents building/running on 10.13 to 11.15 versions on MacOS? I would very much like to use version Textadept v. 12 on Mojave.

**Comment by orbitalquark** [comment]
Qt 6 officially supports macOS >= 11. There is nothing stopping you from trying to build/run Textadept on macOS 10.13-10.15. Let us know if it works!

**Comment by cbertelli** [comment]
Reading the Qt 6 docs (https://doc.qt.io/qt-6/macos.html#supported-versions) I see that the requirement is XCode v. 13 which is not compatible with 10.13 (I think it runs on 10.14).

**Comment by orbitalquark** [comment]
I wonder if that's just for compiling Qt 6 from source, as opposed to just sourcing the headers and linking to existing libraries. You can also edit the CMakeLists.txt to compile with Qt 5 instead of 6 on macOS.
